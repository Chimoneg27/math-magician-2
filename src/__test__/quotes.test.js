import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/quotes';

describe('Quotes', () => {
  test('Should be the same as the snapshot', () => {
    const component = renderer.create(<Quotes />);
    const instance = component.toJSON();
    expect(instance).toMatchSnapshot();
  });
});
