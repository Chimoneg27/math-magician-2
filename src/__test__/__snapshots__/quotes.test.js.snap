// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes Should be the same as the snapshot 1`] = `
<div
  className="quotes-box"
>
  <blockquote>
    <span
      className="quote"
    />
    <cite />
  </blockquote>
</div>
`;
